src/BenchmarksDriver2/benchmarks.proxy.yml

```diff
@@ -161,6 +186,7 @@ profiles:
       downstreamUri: http://10.0.0.103:5000
       downstreamAddress: 10.0.0.103
       downstreamPort: 5000
+      proxyUri: http://asp-citrine-lin:5001
```

Comment:
Same as before. Also use the 10.0.0.102 address as this is the ip on the private vlan for these machines, otherwise it it goes through NIC (1GB)